etl/steps/data/garden/energy/2023-12-12/photovoltaic_cost_and_capacity.meta.yml

```diff
@@ -1,43 +1,36 @@
-
-    According to Farmer & Lafond (2016), the data are mostly taken from the [Santa-Fe Performance Curve DataBase](https://pcdb.santafe.edu/). The database has been constructed from personal communications and from [Colpier and Cornland (2002)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0095), [Goldemberg et al. (2004)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0130), [Lieberman (1984)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0180), [Lipman and Sperling (1999)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0190), [Zhao (1999)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0310), [McDonald and Schrattenholzer (2001)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0205), [Neij et al. (2003)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0235), [Moore (2006)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0215), [Nemet (2006)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0240), [Schilling and Esmundo (2009)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0265). The data on photovoltaic prices has been collected from public releases of Strategies Unlimited, Navigant and SPV Market Research. The data on nuclear energy is from [Koomey and Hultman (2007)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0165) and [Cooper (2009)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0100). The DNA sequencing data is from [Wetterstrand (2015)](https://www.sciencedirect.com/science/article/pii/S0048733315001699#bib0290) (cost per human-size genome), and for each year the last available month (September for 2001-2002 and October afterwards) was taken and corrected for inflation using the US GDP deflator.
-
-    Prices from Farmer & Lafond (2016) have been converted to 2021 US$ using the US GDP deflator: https://www.multpl.com/gdp-deflator/table/by-year
-
-    Photovoltaic capacity data between 2004 and 2021 has been taken from IRENA.
-
-    Photovoltaic cost data between 2010 and 2021 has been taken from IRENA.
 tables:
   photovoltaic_cost_and_capacity:
+    title: Solar photovoltaic cost and capacity
     variables:
       cost:
         title: Solar photovoltaic module price
         short_unit: $/W
         unit: 2021 US$ per Watt
-        description: |
-          Global average price of solar photovoltaic modules.
-
-          IRENA presents solar PV module price series for a number of different module technologies. Here we have adopted the series for thin film a-Si/u-Si or Global Index (from Q4 2013).
+        description_key:
+          - IRENA presents solar PV module price series for a number of different module technologies. Here we use the series for thin film a-Si/u-Si or Global Index (from Q4 2013).
```

Comment:
this description_key tells people what metric you've used but not what it actually is - e.g., I have no idea what `thin film a-Si/u-Si or Global Index (from Q4 2013)` is.